### PR TITLE
MAT-2712 Parse FHIR SDEs

### PR DESCRIPTION
**Pull request by serhii-ilin**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Review by AndrewBird81 — APPROVED** [review]

**Review by adongare — APPROVED** [review]

File: lib/measure-loader/bundle_loader.rb [hunk 1]
```diff
@@ -246,6 +248,19 @@ def parse_population_sets(fhir_measure, measure_lib_name)
...
+        fhir_measure.supplementalData&.each do |sd|
+          supplemental_data_elements << CQM::StatementReference.new(
+            library_name: measure_lib_name,
+            statement_name: sd.criteria.expression.value
+          )
+        end
+      end
+      supplemental_data_elements
+    end
+
```
Comment (adongare) [inline lib/measure-loader/bundle_loader.rb:263]: 🎉
